### PR TITLE
fix: adding conditions for tvOS location providers

### PR DESCRIPTION
**Pull request by vdimitrovv**
Adding conditions to only check if location services is enabled, because tvOS doesn't support `significantLocationChangeMonitoringAvailable`, `headingAvailable` and `isRangingAvailable`.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| tvOS     |    ✅     |

## Checklist
* [x] I have tested this on a device/simulator for each compatible OS
* [x] I mentioned this change in `CHANGELOG.md`

### CODE REVIEW

**Comment by mikehardy** [comment]
Supporting location and power information in combination with tvOS is difficult! This is the 2nd or 3rd snag we have had there. @markhomoki - you are also an active member here using tvOS, does this look good to you? It looks good to me but I know you have a vested interest...

@vdimitrovv your second fix here (at least, that's just looking at recent changes), thank you 🏆  😄

**Comment by markhomoki** [comment]
This looks good to me apart from the double if statements. Could be done with if/else.

File: ios/RNDeviceInfo/RNDeviceInfo.m [hunk 1]
```diff
@@ -400,12 +400,19 @@ - (NSDictionary *)powerState
 
 RCT_EXPORT_METHOD(getAvailableLocationProviders:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
 {
+#if !TARGET_OS_TV
```
Comment (markhomoki) [inline ios/RNDeviceInfo/RNDeviceInfo.m]: Can you change to if else, please?

Comment (mikehardy) [reply ios/RNDeviceInfo/RNDeviceInfo.m]: Seems reasonable, thanks for the review @markhomoki !

**Comment by mikehardy** [comment]
Haha and you already did it ;-)

**Comment by mikehardy** [comment]
I'll pull these locally and build them just to make sure but I don't see any reason why I won't merge and release shortly. Thanks again to both of you

**Comment by vdimitrovv** [comment]
Change done 👍

For further context, both @markhomoki and I are on the same team :-)

**Comment by mikehardy** [comment]
> Change done 👍
> 
> For further context, both @markhomoki and I are on the same team :-)

Oh well that's just sneaky, you're ganging up on me ;-). I'm really happy we have tvOS users active though - it's a big help as it's not a target I build normally

**Comment by mikehardy** [comment]
Sorry for the delay on this - just did testing and noticed my example app suffered from the same "what's this tvOS thing?" problem, where it just expected battery status etc to be there.

I fixed that and re-pushed to your branch so the PR was good (though please double-check) and I'll publish 2.0.3 in a minute

**Comment by mikehardy** [comment]
...and it's out - seems fine to me in iOS and tvOS but please holler if I messed anything up :-), thanks again

**Pull request merged**